From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sat, 26 Nov 2016 01:32:00 +0200
Subject: Disable running gyp on shared deps

Author: Stephen Gallagher <sgallagh@redhat.com>

Modified 2016-11-26 by Jakub Jirutka <jakub@jirutka.cz> to update for
Node.js 7.2.0

diff --git a/Makefile b/Makefile
index 94013466..de3f5e9a 100644
--- a/Makefile
+++ b/Makefile
@@ -169,7 +169,7 @@ with-code-cache test-code-cache:
 	$(warning '$@' target is a noop)
 
 out/Makefile: config.gypi common.gypi node.gyp \
-	deps/uv/uv.gyp deps/llhttp/llhttp.gyp deps/zlib/zlib.gyp \
+	deps/uv/uv.gyp deps/llhttp/llhttp.gyp \
 	deps/simdutf/simdutf.gyp deps/ada/ada.gyp \
 	tools/v8_gypfiles/toolchain.gypi tools/v8_gypfiles/features.gypi \
 	tools/v8_gypfiles/inspector.gypi tools/v8_gypfiles/v8.gyp
